source/main.cc

```diff
@@ -386,6 +387,7 @@ void print_version_information(Stream &stream)
   stream << "Version information of underlying libraries:\n"
          << "   . deal.II:    "
          << DEAL_II_PACKAGE_VERSION << '\n'
+         << DEAL_II_GIT_SHORTREV    << '\n'
```

Comment:
This isn't going to look good -- you will have
```
   . deal.II:     8.5.0
45678abcdf
```
because you have two `\n` here. Please check how this looks and create a new pull request that fixes the output. (I would also put the hash into parentheses after the revision number.)